src/renderer/extensions/vueNodes/composables/useNodePointerInteractions.ts

```diff
@@ -6,12 +6,34 @@ import { useVueNodeLifecycle } from '@/composables/graph/useVueNodeLifecycle'
 import { useCanvasInteractions } from '@/renderer/core/canvas/useCanvasInteractions'
 import { layoutStore } from '@/renderer/core/layout/store/layoutStore'
 import { useNodeEventHandlers } from '@/renderer/extensions/vueNodes/composables/useNodeEventHandlers'
-import { isMultiSelectKey } from '@/renderer/extensions/vueNodes/utils/selectionUtils'
 import { useNodeDrag } from '@/renderer/extensions/vueNodes/layout/useNodeDrag'
+import { isMultiSelectKey } from '@/renderer/extensions/vueNodes/utils/selectionUtils'
+import { app } from '@/scripts/app'
+
+// Forward keydown events to litegraph's processKey when Vue nodes have focus
+let keydownForwardingInitialized = false
+
+function initKeydownForwarding() {
+  if (keydownForwardingInitialized) return
+  keydownForwardingInitialized = true
+
+  document.addEventListener(
+    'keydown',
+    (e) => {
+      const canvas = app.canvas
+      if (!canvas) return
+      if (e.target === canvas.canvas) return
+      canvas.processKey(e)
+    },
+    true
+  )
+}
```

Comment:
I don't want to add more arbitrary event listeners in various composables if we can help it.
Could you use [this](https://vueuse.org/core/useMagicKeys/#usemagickeys)?